submissions/iakow/friends-app/js/main.js

```diff
@@ -0,0 +1,232 @@
+"use strict";
+
+disableFilters();
+
+const USERS_AMOUNT = 100;
+
+mountUserList(getPlaceholders());
+
+const ALL_USER_CARDS = [];
+
+fetch(`https://randomuser.me/api/?results=${USERS_AMOUNT}`)
+  .then((response) => response.json())
+  .then((response) => {
+    response.results.forEach((userData) => {
+      ALL_USER_CARDS.push(getUserCard(userData));
+    });
+
+    mountUserList(ALL_USER_CARDS);
+    enableFilters();
+  });
+
+addHandlers();
+
+
+function addHandlers() {
```

Comment:
A lot of if-else statements. Not readable. Change code please. Place in separate functions or something else.
Not try to do all logic in one function. With this approach when your app will be bigger your code will be like spaghetti =(